modules/@ergo/users/components/Grid/Layout/Table/Cells/Data/GridCheckDataCell.vue

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
Is this file is in the correct place? All components related with cell data are included in @core module.

Reply:
It's releated to @users only